doc/cephadm/services/rgw.rst

```diff
@@ -140,43 +140,98 @@ See also :ref:`multisite`.
 Setting up HTTPS
 ----------------
 
-In order to enable HTTPS for RGW services, apply a spec file following this scheme:
+RGW services as other cephadm managed services support three ways of configuring
+HTTPS certificates, all managed through the cephadm Certificate Manager (certmgr):
+
+- **cephadm-signed (default):**
+  If ``ssl`` is set to true but no certificate is specified, cephadm generates and
+  signs a certificate for the RGW service automatically.
+
+- **inline:**
+  User can set the ``certificate_source`` to ``inline`` in the spec and
+  embed the certificate and private key directly in the spec using
+  the ``ssl_cert`` and ``ssl_key`` fields.
+
+- **reference:**
+  User can register his own certificate and key with certmgr and
```

Comment:
We seem to prefer a plural with `they`. This perhaps clashes a bit with `the user` in the later sections so should you take on this suggestion I'd change the `Option 3` text also.

```suggestion
  Users can register their own certificate and key with certmgr and
```